<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>568</width>
    <height>599</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>275</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>270</height>
       </size>
      </property>
      <property name="title">
       <string>Files</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>1</number>
       </property>
       <property name="leftMargin">
        <number>1</number>
       </property>
       <property name="topMargin">
        <number>1</number>
       </property>
       <property name="rightMargin">
        <number>1</number>
       </property>
       <property name="bottomMargin">
        <number>1</number>
       </property>
       <item>
        <widget class="QListWidget" name="listFiles">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::MultiSelection</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>65</height>
          </size>
         </property>
         <widget class="QToolButton" name="btnAddFile">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>24</width>
            <height>25</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="bin64ed.qrc">
            <normaloff>:/images/add.png</normaloff>:/images/add.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>18</width>
            <height>18</height>
           </size>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonIconOnly</enum>
          </property>
         </widget>
         <widget class="QPushButton" name="btnGoFile">
          <property name="geometry">
           <rect>
            <x>480</x>
            <y>10</y>
            <width>75</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>PushButton</string>
          </property>
         </widget>
         <widget class="QToolButton" name="btnDeleteFile">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>0</y>
            <width>24</width>
            <height>24</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="bin64ed.qrc">
            <normaloff>:/images/remove.png</normaloff>:/images/remove.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>18</width>
            <height>18</height>
           </size>
          </property>
         </widget>
         <widget class="QToolButton" name="btnClearList">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>0</y>
            <width>24</width>
            <height>24</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="bin64ed.qrc">
            <normaloff>:/images/clear.png</normaloff>:/images/clear.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>18</width>
            <height>18</height>
           </size>
          </property>
         </widget>
         <widget class="QRadioButton" name="checkEncode">
          <property name="geometry">
           <rect>
            <x>180</x>
            <y>10</y>
            <width>82</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>E&amp;ncode</string>
          </property>
         </widget>
         <widget class="QRadioButton" name="checkDecode">
          <property name="geometry">
           <rect>
            <x>270</x>
            <y>10</y>
            <width>82</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Decode</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="textOutputDirectory">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>45</y>
            <width>291</width>
            <height>20</height>
           </rect>
          </property>
         </widget>
         <widget class="QToolButton" name="btnPickDir">
          <property name="geometry">
           <rect>
            <x>290</x>
            <y>45</y>
            <width>25</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>30</y>
            <width>101</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Output directory</string>
          </property>
         </widget>
         <widget class="QCheckBox" name="checkImgTag">
          <property name="geometry">
           <rect>
            <x>340</x>
            <y>45</y>
            <width>211</width>
            <height>19</height>
           </rect>
          </property>
          <property name="text">
           <string>Output as HTML img tags</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Text</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QWidget" name="widget_2" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
         <widget class="QRadioButton" name="checkEncodeText">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>0</y>
            <width>82</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Encode</string>
          </property>
         </widget>
         <widget class="QRadioButton" name="checkDecodeText">
          <property name="geometry">
           <rect>
            <x>110</x>
            <y>0</y>
            <width>82</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Decode</string>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="textInput">
         <property name="placeholderText">
          <string>Enter text..</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="textOutput">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_3" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>35</height>
          </size>
         </property>
         <widget class="QPushButton" name="btnGoText">
          <property name="geometry">
           <rect>
            <x>470</x>
            <y>10</y>
            <width>75</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>PushButton</string>
          </property>
         </widget>
         <widget class="QPushButton" name="btnClearText">
          <property name="geometry">
           <rect>
            <x>390</x>
            <y>10</y>
            <width>75</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>Clear</string>
          </property>
         </widget>
         <widget class="QToolButton" name="btnAbout">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>10</y>
            <width>25</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="bin64ed.qrc">
            <normaloff>:/images/info_icon.png</normaloff>:/images/info_icon.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>19</width>
            <height>19</height>
           </size>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="bin64ed.qrc"/>
 </resources>
 <connections/>
</ui>
